src/connector/src/sink/formatter/mod.rs

```diff
@@ -107,6 +109,70 @@ impl SinkFormatterImpl {
             )))
         }
     }
+
+    pub fn new_with_redis(
```

Comment:
The syntax for create redis sink with key value template format may follow the new `create sink ... format XXX encode XXX` syntax.

If this PR is urgent, we can move the template row encoder to a separate PR, and in this PR we only use the json encoder for both key and value.

Reply:
Yes, I will create a new pull request to implement the new syntax. I think we can initially implement it in the current way, and then create a new pull request for modifications.